src/macros.rs

```diff
@@ -32,6 +32,7 @@ macro_rules! vertex_attr_array {
 
 // For internal usage
 #[macro_export]
+#[doc(hidden)]
```

Comment:
do we have to hide that? could be useful to people

Reply:
Nah, I just thought it wasn't intended to be used due to the comment. If we want to keep it, I'll give it some simple docs.